docs/users/cli-usage.md

```diff
@@ -182,6 +182,66 @@ Successfully started example-backdrop-site
 Your application can be reached at: http://example-backdrop-site.ddev.local
 ```
 
+### Flow Framework Quickstart
+
+To get started with flow Framework or neos, clone **your** project repository and navigate to the project directory.
+
+```
+git clone https://github.com/example-user/example-neos-site
+cd example-backdrop-site
+composer install
+```
+
+To set up ddev for your project, enter the command:
+
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a project name, docroot, and project type. Use **php** as project type._
+
+After `ddev config`, you're nearly ready to start running your project.
+
+We need to point flow to the correct database. To do that please create `Configuration/Development/Ddev/Settings.yaml`:
+
+```
+TYPO3:
+  Flow:
+    utility:
+      environment:
+        # Defines the base directory which Flow may use for storing different kinds
+        # of temporary files.
+        # The directory must be writable and Flow will automatically create a sub
+        # directory (named after the context) which will contain the actual temporary files.
+        temporaryDirectoryBase: '/tmp/FLOW'
+    persistence:
+      backendOptions:
+        host:     'db'
+        dbname:   'db'  # adjust to your database name
+        user:     'db'  # adjust to your database user
+        password: 'db'  # adjust to your database password
+        port:     3306
+      doctrine:
+        sqlLogger: 'TYPO3\Flow\Persistence\Doctrine\Logging\SqlLogger'
+
+```
+
+Now you could run the project, sadly the `FLOW_CONTEXT` env var is not set yet.
+You can achieve that by editing the `.ddev/docker-compose.yaml` and by changing the `.ddev/nginx-site.conf
+
+Run ddev using and you are good to go:
+
+```
+ddev start
+```
+
+When `ddev start` runs, it outputs status messages to indicate the project environment is starting. When the startup is complete, ddev outputs a message like the one below with a link to access your project in a browser.
+
+```
+Successfully started example-backdrop-site
```

Comment:
Make sure to update from "backdrop" to the actual names you use in your example.

Reply:
ok, will chech that again